jwt/jwt.go

```diff
@@ -0,0 +1,37 @@
+package jwt
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+)
+
+var secretKey = []byte("farawin")
+
+func CreateToken(id string) (string, error) {
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
+		"id":  id,
+		"exp": time.Now().Add(time.Hour * 24).Unix(),
```

Comment:
Also you can create two constants to map the keys here. I mean you may have problem to read `"exp"` in other places(like in middleware). you can simplify it by using a constant like `const TokenExpireTime="exp"` and then pass that consts around to simplify you mappings.